src/runtime/SharedArrayBufferObject.cpp

```diff
@@ -98,8 +96,7 @@ void* SharedArrayBufferObject::operator new(size_t size)
     static MAY_THREAD_LOCAL GC_descr descr;
     if (!typeInited) {
         GC_word obj_bitmap[GC_BITMAP_SIZE(SharedArrayBufferObject)] = { 0 };
-        Object::fillGCDescriptor(obj_bitmap);
-        GC_set_bit(obj_bitmap, GC_WORD_OFFSET(SharedArrayBufferObject, m_backingStore));
+        SharedArrayBufferObject::fillGCDescriptor(obj_bitmap);
```

Comment:
Use SharedArrayBufferObject is intened operation
user can new variable into SharedArrayBuffer and override fillGCDescriptor.

Reply:
Even if the user is able to do it, it's not intuitive and it's inconsistent with other implementations. If you want to keep this change, it would be nice to comment to describe your intention a bit more kindly.

Reply:
I agree to conflict with other implementations. 
so I change code